applications/main/lfrfid/lfrfid.c

```diff
@@ -249,31 +247,26 @@ bool lfrfid_delete_key(LfRfid* app) {
 }
 
 bool lfrfid_load_key_data(LfRfid* app, FuriString* path, bool show_dialog) {
-    bool result = false;
-
-    do {
-        app->protocol_id = lfrfid_dict_file_load(app->dict, furi_string_get_cstr(path));
```

Comment:
Removing this causing crash

Reply:
Indeed I missed the update of app->protocol_id. My bad.